src/Engine/ProtoCore/RuntimeData.cs

```diff
@@ -55,25 +48,39 @@ public RuntimeData()
         /// <param name="core"></param>
         /// <param name="uid"></param>
         /// <returns></returns>
-        public CallSite GetCallSite(GraphNode graphNode,
-                                    int classScope,
-                                    string methodName,
-                                    Executable executable,
-                                    int runningBlock,
-                                    Options options,
-                                    RuntimeStatus runtimeStatus
-             )
+        public CallSite GetCallSite(int classScope, string methodName, Executable executable, RuntimeCore runtimeCore)
```

Comment:
Both from the VM runtime - one is from instruction CALLR's handler, the other one is builtin function `DotMethod()`.